subsys/bluetooth/common/Kconfig

```diff
@@ -150,6 +150,7 @@ config BT_BUF_EVT_RX_COUNT
 
 config BT_BUF_EVT_DISCARDABLE_SIZE
 	int "Maximum supported discardable HCI Event buffer length"
+	depends on !BT_HCI_RAW
```

Comment:
What's the reason these are in `common/` instead of `host/`? If they're host specific they should probably be moved, and then you don't need the explicit exclusion of HCI_RAW.

Reply:
Agree. Sure will give a try, and get back.